docs/src/howto-guides/026_how_to_install_thin_edge_manually.md

```diff
@@ -0,0 +1,224 @@
+For a minimal configuration of thin-edge.io with Cumulocity IoT, you will need to download:
+
+- tedge\_{VERSION}\_amd64.deb
+- tedge\_agent\_{VERSION}\_amd64.deb
+- tedge\_mapper\_{VERSION}\_amd64.deb
+
+<p align="center">
+  <img src="./images/manual_installation-minimum_deb_packages.png" alt="Sublime's custom image"/>
+</p>
+
+Next, unpack each deb file and copy the binary to `/usr/bin`.
+For `tedge` debian package do:
+
+```shell
+ar -x tedge_*_amd64.deb | tar -xf data.tar.xz
+```
+This unpacks two directories `usr/bin/`, move its contents to `/usr/bin`
+
+```shell
+sudo mv usr/bin/tedge /usr/bin
+```
+
+> Note: Do the same for tedge\_agent and tedge\_mapper debian packages.
+
+
+## Step 1: Creating the tedge user
+
+The next step is to create the tedge user. This is normally taken care by the debian package for the `tedge` CLI tool.
+
+To do this in Gentoo, for example, you can:
+
+```shell
+sudo groupadd --system tedge
+
+sudo useradd --system --no-create-home -c "" -s /sbin/nologin -g tedge tedge
+```
+
+Now that we have created the tedge user, we need to allow the tedge user to call commands with `sudo` without requiring a password:
+
+```shell
+sudo echo "tedge  ALL = (ALL) NOPASSWD: /usr/bin/tedge, /etc/tedge/sm-plugins/[a-zA-Z0-9]*, /bin/sync, /sbin/init" >/etc/sudoers.d/tedge
+```
+
+Next, create the files and directories required by thin-edge.io and restart mosquitto too.
+
+```shell
+sudo rc-service mosquitto stop
+sudo tedge --init
+sudo rc-service mosquitto start
+sudo tedge_agent --init
+sudo tedge_mapper --init c8y
+```
+
+This should show the following output:
+
+<p align="center">
+  <img src="./images/manual_installation-binaries_init.png" alt="Sublime's custom image"/>
+</p>
+
+> Note: if you do not restart mosquitto you will see a Connection refused error. Do not worry, this error can be ignored.
+
+Ensure that running the init has created the following files and directories in `/etc/tedge`:
+
+<p align="center">
+  <img src="./images/manual_installation-tedge_directories.png" alt="Sublime's custom image"/>
+</p>
+
+## Step 3: Creating mosquitto bridge
+
+To create the mosquitto bridge simply run:
+
+```shell
+sudo echo "include_dir /etc/tedge/mosquitto-conf" >> /etc/mosquitto/mosquitto.conf
+```
+You can test that `mosquitto` works by running: 
+
+```shell
+sudo mosquitto --config-file /etc/mosquitto/mosquitto.conf
+```
+
+## Step 4: Creating OpenRC service files
+
+You will need service files for tedge\_agent and tedge\_mapper. For example:
+
+> Note that, for Cumulocity IoT, the `tedge connect` command expects three service files called: mosquitto, tedge-agent and tedge-mapper-c8y
+
+For the `tedge-agent` service an example file is the following:
+> FILE: /etc/init.d/tedge-agent
+```sh
+#!/sbin/runscript
+
+start() {
+   ebegin "Starting tedge-agent"
+   start-stop-daemon --start --background --exec tedge_agent
```

Comment:
`tedge_agent` and tedge_mapper <cloud>` should be started as `tedge` user. Where did you define the user of the processes?